cdi2/src/test/resources/META-INF/beans.xml

```diff
@@ -1,7 +1,10 @@
-<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xmlns="http://java.sun.com/xml/ns/javaee"
-       xsi:schemaLocation="
-      http://java.sun.com/xml/ns/javaee
-      http://java.sun.com/xml/ns/javaee/beans_1_0.xsd">
-
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_2_0.xsd"
+    bean-discovery-mode="all" version="2.0">
+    <!-- Exclude the class to force it to be loaded as an unmanaged dependency -->
```

Comment:
Would it be possible to test both the managed and unmanaged scenarios explicitly?

Reply:
I don't think it would be possible in a single maven project, but there are 2 step classes in the project and only 1 is excluded from the CDI scan so we already have 1 managed step class and 1 unmanaged step class.

Otherwise I think we would need to programatically remove the bean from the CDI container before the test is started, I'm not sure if it is even feasible.

Reply:
@dcendents keeping this beans.xml you can split the step in 2 and have one exclude and the other not excluded, should enable to test both cases at once no?

Reply:
I see what you mean, I added a new UnmanagedBellyStepDefinitions.

Reply:
Cheers. I push a bit more clean up so it's more apparent how tests and example should be written.